src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -86,6 +88,29 @@ public static bool ZipDeploymentAppSettingsExist(this IEnvironment environment)
                    !string.IsNullOrEmpty(environment.GetEnvironmentVariable(ScmRunFromPackage));
         }
 
+        public static bool AzureFilesAppSettingsExist(this IEnvironment environment)
```

Comment:
Just a note for later, eventually it will be possible to have azure files enabled without explicitly having these settings when running in secretless mode. for now we will return false for such apps even though it might technically possible to edit (which is fine).